### ISSUE TITLE
multiple grid on same page and filtering

### ISSUE THREAD

**Issue by codeweb** [joeharrison714/MVCGrid.Net#98]
Scenario: two grids on the same page (prefixes already configured) and one filter which must filter a column present in both grid. The parameter name in the url will take the name of the first grid.

**Comment by joeharrison714** [comment]
try calling `MVCGrid.setFilters` on both grids when you want to trigger the filtering.

```        $('#filterButton').click(function () {
            MVCGrid.setFilters('Grid1', { LastName: 'Smith' })
            MVCGrid.setFilters('Grid2', { LastName: 'Smith' })
        });```

**Comment by codeweb** [comment]
No, in that way i will have duplicate parameters because i have grid with prefix. Moreover some filters are not column of the grid (so i need AdditionalQueryOption). Why can't i use in a grid AdditionalQueryOption of the other grid (with prefix)?
